mkdocs.yml

```diff
@@ -152,6 +152,8 @@ nav:
             - Interpolated Sets:                     tutorials/materials/interpolated-sets.md
             - Molecule on a Surface:                 tutorials/materials/molecule-surface.md
             - Interface:                             tutorials/materials/slabs-interface.md
+        - Python Packages:
```

Comment:
Probably fits under "General Functionality", no?

Reply:
Fixed in recent commit